.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,7 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: true
```

Comment:
maybe no blank issues dont you think ? it's either go to the chat, either use a template, or we'll end up again with issues partially filled

Reply:
Good point, I must admit I copy and pasted this part.